pkg/vdr/sov/vdr.go

```diff
@@ -0,0 +1,155 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package sov
```

Comment:
@Moopli - no - this is not did:sov. It is confusing to have this package named sov in vdr.

Additionally, it doesn't really help. You need to fix the document in did exchange.

Reply:
Let's remove this change from this PR @Moopli